website/blog/2018-06-25-on-consuming-and-publishing-es6+-packages.md

```diff
@@ -0,0 +1,218 @@
+---
+layout: post
+title:  "On Consuming (and Publishing) ES6+ Packages"
```

Comment:
You switch between use of "ES6" and "ES2015" a few times during the article. Pick one, add the other in brackets after the first occurrence, then find-replace the rest.

Reply:
Given that you mention preset-es2015 and publishing under es2015/etc, my vote is for 2015. 
It also lumps all the modern language levels under one title type, which will make the mental model clearer vs es5.